android/src/main/AndroidManifestNew.xml

```diff
@@ -0,0 +1,5 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
Why the two manifest files?

Reply:
There is a requirement [from RN](https://github.com/react-native-community/discussions-and-proposals/issues/671) to have a namespace, but not all Gradle versions do support it. Therefore within the `build.gradle` there is a check, if namespace is supported within gradle, then sets the namespace in `build.gradle` and use `AndroidManifestNew.xml` without namespace. Otherwise the other way around. This part is completely taken from the scaffold.